docs/mana_proposal.md

```diff
@@ -0,0 +1,100 @@
+# Mana Implementation Proposal
+
+The goal of this document is to provide a high level overview of how mana will be implemented in GoShimmer.
+
+## Introduction
+
+Mana is a a reputation system for nodes within the IOTA network.
+
+Reputation is gained by contributing to the network, i.e. creating value transfers.
+Reputation is lost when not cooperating with the network: using up all the available bandwidth, advocating for
+"bad" transfers during voting, etc..
+
+As time passes, earned mana of a node decays to encourage keeping up the good behavior.
+
+## Scope
+
+The sope of the first implementation of mana into GoShimmer is to verify that mana calculations work,
+study mana distribution in the test network and verify that nodes have similar view on the network. A test dApp could
+be written to see how nodes react to "simulated misbehaving" nodes.
+
+## Mana Calculation
+
+Mana is esentially the reputation score of a node in the IOTA network. Mana is calculated locally in each node, as a
+function that takes value transactions as input and produces the Base Mana Vector as output.
+It consists of Base Mana 1 and Base Mana 2. Given a value transaction, these are determined as follows:
+ 1. Base Mana 1 is revoked from the node that created the output(s) used as input(s) in the transaction, and is pledged to
+    the node creating the new output(s).
+ 2. Base Mana 2 is freshly created at the issuance time of the transaction, awarded to the node, but decays with time.
+
+Access and Consensus Mana (mana used by other node modules) are derived from Base Mana 1 and 2 and "smoothened" through
```

Comment:
At least to me, it's easier to think like this:

Each Eff Base Mana is the EMA of a Base Mana
Access and Consensus Mana are derived from Eff Base Mana 1 and 2